build.fsx

```diff
@@ -1,22 +0,0 @@
-#r @"packages/FAKE/tools/FakeLib.dll"
```

Comment:
Now that `build.fsx`, `build.sh` and `build.cmd` are removed, how should we tell to the users to build the source locally from the command-line? (So we can put it on the README.)

Reply:
I guess they would have to do similar with `travis.yml`?

```
./validate.sh
dotnet build -c Release
dotnet test -c Release --no-build tests/Hedgehog.Tests/Hedgehog.Tests.fsproj
dotnet test -c Release --no-build tests/Hedgehog.CSharp.Tests/Hedgehog.CSharp.Tests.csproj
```

And then, how can we build the NuGet package in order to push it on NuGet Gallery?

Reply:
Will update the README.